src/commands/set.js

```diff
@@ -68,39 +75,47 @@ const attemptWriteToFile = (variable, value) => {
 	return result;
 };
 
-const setBoolean = (variable, path) => value => {
+const setBoolean = dotNotation => value => {
```

Comment:
what is the reason of `dotNotation => value`?
would it be more simple if you do like `(dotNotation, value) =>`?

Reply:
The HOF was a hangover from the previous implementation, but can be removed in this PR with the simplifications that are being put in place.